### PR TITLE
Fix up the readme.

### PR DESCRIPTION
**Pull request by NikolaBorisov**
Adding init example. Listing some of the features.

### CODE REVIEW

**Review by vipin211 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -4,31 +4,42 @@ Resources used in the building of ecredis:
...
 - https://github.com/Nordix/eredis_cluster
 
+ecredis is a high performance Erlang client for Redis Cluster. It uses eredis as the underlying
+implementation. Allows connecting to multiple redis clusters.
+
+## Features
+
+* Allows connections to multiple redis clusters at the same time
+* High throughput. The requests are executed directly on the callers process.
+
```
Comment (balusu-murali) [inline README.md:14]: should we mention that it does not work with a single client.

**Review by balusu-murali — APPROVED** [review]

**Review by mdonohue — APPROVED** [review]
How will people discover this repo?

**Comment by NikolaBorisov** [comment]
> How will people discover this repo?

We can try to add it to this list: https://redis.io/clients#erlang

File: README.md (hunk 1 above)
Comment (NikolaBorisov) [reply README.md:14]: Do you mean that it doesn't work for non clustered redis?

Comment (NikolaBorisov) [reply README.md:14]: I think it actually should because we have this function: get_cluster_slots_from_single_node(Node);

**Pull request merged**